### ISSUE TITLE
Exception when scanning Azure subscription

### ISSUE THREAD

**Issue by Vinko90** [mondoohq/client#157]
**Describe the bug**
A `panic: interface conversion: interface {} is nil, not string` exception was reported when using mondoo-cli to scan an Azure subscription. Exception thrown at 14% complete, and partial report is uploaded to my fleet dashboard.

**To Reproduce**
1. az login
2. mondoo scan -t az://subscriptions/{subscriptionId}
3. Wait for scan to complete

**Expected behavior**
Wait for the final report, like mentioned in documentation: https://mondoo.com/docs/cloud/azure/#scan-azure-subscription

**Screenshots**
Here is the complete Exception:
```bash
panic: interface conversion: interface {} is nil, not string

goroutine 128 [running]:
go.mondoo.io/mondoo/lumi/resources.(*lumiAzurermWebAppsite).GetStack(0xc000ff8b28)                                                                                                                                                                        /builds/mondoolabs/mondoo/lumi/resources/azure_rm_web.go:530 +0x986
go.mondoo.io/mondoo/lumi/resources.(*lumiAzurermWebAppsite).ComputeStack(0xc000ff8b28)
/builds/mondoolabs/mondoo/lumi/resources/core.lr.go:74891 +0x55
                                                                                                                                                                               go.mondoo.io/mondoo/lumi/resources.(*lumiAzurermWebAppsite).Compute(0xc000713a00?, {0xc00190a348, 0x5})
/builds/mondoolabs/mondoo/lumi/resources/core.lr.go:74809 +0x17a
go.mondoo.io/mondoo/lumi.(*Runtime).WatchAndUpdate(0xc001610d20, {0x84816c8, 0xc000ff8b28}, {0xc00190a348, 0x5}, {0xc000b29300, 0x31}, 0xc000eed0b0)
/builds/mondoolabs/mondoo/lumi/runtime.go:284 +0x39b
                                                                                                                                    go.mondoo.io/mondoo/llx.runResourceFunction(0xc0002f0380, 0xc001f92840, 0xc001907680, 0x200000002)
/builds/mondoolabs/mondoo/llx/builtin.go:691 +0x22f
go.mondoo.io/mondoo/llx.(*blockExecutor).runBoundFunction(0xc0002f0380, 0xc001f92840, 0xc001907680, 0xc001ac2c70?)
                                                                                                                                                                                                /builds/mondoolabs/mondoo/llx/builtin.go:774 +0x10d
go.mondoo.io/mondoo/llx.(*blockExecutor).runFunction(0xc0002f0380, 0xc001907680, 0xc001421548?)
/builds/mondoolabs/mondoo/llx/llx.go:703 +0x111
                                                                                                                                                                                       go.mondoo.io/mondoo/llx.(*blockExecutor).runChunk(0xc0002f0380, 0xc001ac2d88?, 0x4b3be2c?)
/builds/mondoolabs/mondoo/llx/llx.go:720 +0x2c5
go.mondoo.io/mondoo/llx.(*blockExecutor).runRef(0xc000eecd80?, 0x6451f00?)
                                                                                                                                                                                                /builds/mondoolabs/mondoo/llx/llx.go:745 +0xef
go.mondoo.io/mondoo/llx.(*blockExecutor).runChain(0xc0002f0380, 0x72338bf?)
/builds/mondoolabs/mondoo/llx/llx.go:777 +0xd4
go.mondoo.io/mondoo/llx.(*blockExecutor).run(0xc0002f0380)
                                                                                                                                                                                                                 /builds/mondoolabs/mondoo/llx/llx.go:333 +0x2da
go.mondoo.io/mondoo/llx.(*blockExecutor).runFunctionBlock(0xc001f3a700, {0xc000ff8e98, 0x1, 0xc001f929c0?}, 0x200000000, 0x0?)
                                                                                                                                                                                /builds/mondoolabs/mondoo/llx/llx.go:524 +0x157
go.mondoo.io/mondoo/llx.(*blockExecutor).runFunctionBlocks(0xc001f3a700, {0xc0018f9680, 0xd, 0x6b918a0?}, 0xc0006aa1b0?, 0x1901f924b0?)
                                                                                                                                                        /builds/mondoolabs/mondoo/llx/llx.go:494 +0xfd
                                                                                                                                                                                                      go.mondoo.io/mondoo/llx.arrayBlockListV2(0xc001f3a700, 0xc001f92270, 0xc0019075e0, 0x100000003)
/builds/mondoolabs/mondoo/llx/builtin_array.go:140 +0x556
go.mondoo.io/mondoo/llx.(*blockExecutor).runBoundFunction(0xc001f3a700, 0xc001f92270, 0xc0019075e0, 0x2030007?)
/builds/mondoolabs/mondoo/llx/builtin.go:761 +0x13d
go.mondoo.io/mondoo/llx.(*blockExecutor).runFunction(0xc001f3a700, 0xc0019075e0, 0xc001ac31d0?)
                                                                                                                                                                                                        /builds/mondoolabs/mondoo/llx/llx.go:703 +0x111
go.mondoo.io/mondoo/llx.(*blockExecutor).runChunk(0xc001f3a700, 0xc001ac32a8?, 0x4b3be2c?)
/builds/mondoolabs/mondoo/llx/llx.go:720 +0x2c5
                                                                                                                                                                               go.mondoo.io/mondoo/llx.(*blockExecutor).runRef(0xc001f923c0?, 0x6451f00?)
/builds/mondoolabs/mondoo/llx/llx.go:745 +0xef
go.mondoo.io/mondoo/llx.(*blockExecutor).runChain(0xc001f3a700, 0x475669?)
                                                                                                                                                                        /builds/mondoolabs/mondoo/llx/llx.go:777 +0xd4
    go.mondoo.io/mondoo/llx.(*blockExecutor).triggerChain(0xc001f3a700, 0x6451f00?, 0xc001f92270)
/builds/mondoolabs/mondoo/llx/llx.go:847 +0x393
                                                                                                                                                       go.mondoo.io/mondoo/llx.runResourceFunction.func1({0x6384ec0?, 0xc001e266f0}, {0x0?, 0x0})
/builds/mondoolabs/mondoo/llx/builtin.go:710 +0x285
go.mondoo.io/mondoo/lumi.(*Runtime).WatchAndUpdate.func1()
                                                                                                                                                /builds/mondoolabs/mondoo/lumi/runtime.go:254 +0x258
                                                                                                                                                                                                    go.mondoo.io/mondoo/lumi.(*Runtime).WatchAndUpdate(0xc001610d20, {0x8481688, 0xc000702f00}, {0xc00190a32c, 0x4}, {0xc001fbb6c0, 0x31}, 0xc001f925d0)
                                                                                                                                        /builds/mondoolabs/mondoo/lumi/runtime.go:293 +0x3f4
                                                                                                                                                                                            go.mondoo.io/mondoo/llx.runResourceFunction(0xc001f3a700, 0xc001f92510, 0xc001907590, 0x100000002)
/builds/mondoolabs/mondoo/llx/builtin.go:691 +0x22f
go.mondoo.io/mondoo/llx.(*blockExecutor).runBoundFunction(0xc001f3a700, 0xc001f92510, 0xc001907590, 0x359b0bc8?)
/builds/mondoolabs/mondoo/llx/builtin.go:774 +0x10d
go.mondoo.io/mondoo/llx.(*blockExecutor).runFunction(0xc001f3a700, 0xc001907590, 0xc0017b5950?)
                                                                                                                                                                                                /builds/mondoolabs/mondoo/llx/llx.go:703 +0x111
go.mondoo.io/mondoo/llx.(*blockExecutor).runChunk(0xc001f3a700, 0xc0017b5a28?, 0x4b3be2c?)
/builds/mondoolabs/mondoo/llx/llx.go:720 +0x2c5
go.mondoo.io/mondoo/llx.(*blockExecutor).runRef(0xc001f923c0?, 0x6451f00?)
/builds/mondoolabs/mondoo/llx/llx.go:745 +0xef
go.mondoo.io/mondoo/llx.(*blockExecutor).runChain(0xc001f3a700, 0x72338bf?)
                                                                                                                                                                /builds/mondoolabs/mondoo/llx/llx.go:777 +0xd4
                                                                                                                                                                                                              go.mondoo.io/mondoo/llx.(*blockExecutor).run(0xc001f3a700)
/builds/mondoolabs/mondoo/llx/llx.go:333 +0x2da
go.mondoo.io/mondoo/llx.(*LeiseExecutorV2).Run(0xc001903920?)
                                                                                                                                                                        /builds/mondoolabs/mondoo/llx/llx.go:265 +0x59
    go.mondoo.io/mondoo/llx.RunV2(0xb5dc340?, 0x7223989?, 0x0?, 0xc00190a310?)
/builds/mondoolabs/mondoo/llx/run.go:13 +0x34
go.mondoo.io/mondoo/policy/executor/internal.(*executionManager).executeCodeBundle(0xc0019ff1d0, 0xc0017fa460, 0xc00181dee0?, {0x0, 0x0})
/builds/mondoolabs/mondoo/policy/executor/internal/execution_manager.go:166 +0x34f
                                                                                                                                          go.mondoo.io/mondoo/policy/executor/internal.(*executionManager).Start.func1()
        /builds/mondoolabs/mondoo/policy/executor/internal/execution_manager.go:87 +0x1e5
created by go.mondoo.io/mondoo/policy/executor/internal.(*executionManager).Start
                                                                                                                                                                                /builds/mondoolabs/mondoo/policy/executor/internal/execution_manager.go:56 +0x6a
3)
```

**Desktop (please complete the following information):**
 - OS: Windows 10 - Build 19044.1766
 - Browser: Brave [Version 1.40.109 Chromium: 103.0.5060.66 (Official Build) (64-bit)]
 - Mondoo 6.4.0 (5105a893, 2022-06-28T17:22:14Z)

**Additional context**
Scan performed with Policy: CIS Microsoft Azure Foundations Benchmark Level 1 Profile

**Comment by jaym** [comment]
i was able to reproduce this by creating a web app with a container instead of a stack. The metadata is empty in that case:
```
{
  "id": "/subscriptions/d13c8331-50af-4496-b9d2-3ff8e72d22b9/resourceGroups/test_group/providers/Microsoft.Web/sites/jaymtest1/config/metadata",
  "location": "Central US",
  "name": "metadata",
  "properties": {},
  "tags": {},
  "type": "Microsoft.Web/sites/config"
}
```

**Comment by jaym** [comment]
This has been fixed in the last release (6.7.1)

Please let use know if you're still having issues after upgrading
